classes/class-ledyer-callback.php

```diff
@@ -77,7 +77,7 @@ public function register_routes() {
 	 * @return \WP_REST_Response
 	 */
 	public function handle_notification( \WP_REST_Request $request ) {
-		$request_body = json_decode( $request->get_body() );
+		$request_body = $request->get_json_params();
 		$response     = new \WP_REST_Response();
 
 		if ( json_last_error() !== JSON_ERROR_NONE ) {
```

Comment:
Question: Is this needed? Can `$request->get_json_params()` trigger a json error to be generated, wont the return from the method just be `null` in that case?

Reply:
It seems like it will be an empty array, either way, an `empty` check will suffice. I've amended the code accordingly.